seed/building_sync/mappings.py

```diff
@@ -0,0 +1,976 @@
+            },
+            'annual_peak_demand': {
+                'xpath': './ResourceUses/ResourceUse[EnergyResource="Electricity"]/AnnualPeakConsistentUnits',
+                'type': 'value',
+                'value': 'text'
+            },
+            'annual_natural_gas_energy': {
+                'xpath': './ResourceUses/ResourceUse[EnergyResource="Natural gas"]/AnnualFuelUseConsistentUnits',
+                'type': 'value',
+                'value': 'text'
+            },
+            'is_virtual': {
+                'xpath': './ScenarioType/PackageOfMeasures/CalculationMethod/Modeled',
+                'type': 'value',
+                'value': 'exist'
+            },
+            'measure_ids': {
+                'xpath': './ScenarioType/PackageOfMeasures/MeasureIDs',
+                'type': 'list',
+                'items': {
+                    'id': {
+                        'xpath': './MeasureID',
+                        'type': 'value',
+                        'value': '@IDref'
+                    }
+                }
+            },
+            'resource_uses': {
+                'xpath': './ResourceUses/ResourceUse',
+                'type': 'list',
+                'items': {
+                    'source_id': {
+                        'xpath': '.',
+                        'type': 'value',
+                        'value': '@ID',
+                    },
+                    'type': {
+                        'xpath': './EnergyResource',
+                        'type': 'value',
+                        'value': 'text',
+                        'formatter': to_energy_type
+                    },
+                    'units': {
+                        'xpath': './ResourceUnits',
+                        'type': 'value',
+                        'value': 'text'
+                    }
+                }
+            },
+            'time_series': {
+                'xpath': './TimeSeriesData/TimeSeries',
+                'type': 'list',
+                'items': {
+                    'start_time': {
+                        'xpath': './StartTimestamp',
+                        'type': 'value',
+                        'value': 'text',
+                        'formatter': to_datetime
+                    },
+                    'end_time': {
+                        'xpath': './EndTimestamp',
+                        'type': 'value',
+                        'value': 'text',
+                        'formatter': to_datetime
+                    },
+                    'reading': {
+                        'xpath': './IntervalReading',
+                        'type': 'value',
+                        'value': 'text'
+                    },
+                    'source_id': {
+                        'xpath': './ResourceUseID',
+                        'type': 'value',
+                        'value': '@IDref'
+                    }
+                }
+            }
+        }
+    }
+}
+
+BASE_MAPPING_V2_ALT = {
```

Comment:
can we just name this V2 and deprecate to V2_ALT?

Reply:
The problem is the different locations for some address info, this is why there are two mappings. e.g. some files put address_line_1 at `./Address/StreetAddressDetail/Simplified/StreetAddress`, while others have it at `./Buildings/Building/Address/StreetAddressDetail/Simplified/StreetAddress`. I guess this is part of a broader question of how we want to handle these slight but predictable structural differences. Something worth discussing on Thursday

Reply:
okay, let's discuss Thursday.

Reply:
After discussing, we found out that the current versions in BuidingSync were actually v2.0-pr1. The mapping names have been updated accordingly.